packages/sources/snowflake/src/endpoint/covid-cases.ts

```diff
@@ -0,0 +1,63 @@
+import { Requester, Validator } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/types'
+import { buildSnowflakeJWT } from '../util'
+
+export const supportedEndpoints = ['covid-cases']
+
+export const inputParameters: InputParameters = {
+  state: ['state'],
+  county: ['county'],
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, _, config) => {
+  const validator = new Validator(request, inputParameters)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const state = validator.validated.data.state
+  const county = validator.validated.data.county
+
+  const jwtToken = buildSnowflakeJWT({
+    privateKey: config.api.privateKey,
+    qualifiedUsername: config.api.qualifiedUsername,
+  })
+
+  const response = await Requester.request({
```

Comment:
If you could create a TypeScript interface for the `ResponseSchema`, you can add the type here like this:

```suggestion
  const response = await Requester.request<ResponseSchema>({
```

That adds some more type safety and we can export that type out of this adapter if needed